cypress/utils/utils.ts

```diff
@@ -629,8 +629,6 @@ export function verifyImportErrorMsg(errorMsg: any): void {
 }
 
 export function deleteApplicationTableRows(): void {
-    selectUserPerspective("Developer");
```

Comment:
Removing this from here might fail other TC's that are not calling navigation to application inventory . What do you think ?

Reply:
Actually, @sshveta  we added this line last week in this PR https://github.com/konveyor/tackle-ui-tests/pull/325/files. As I noticed in Import test it was mandatory for the test to reload back Application Inventory for bulk deletion.

Reply:
In this PR I've added the line in after hook to navigate.